Your-Missing-Semester/purple_repo
# T19, T23, T24: routes, sign in and sign up routing, backend server

Latest Commit (most recent commit):
- Used first commit to fix conflicting files and related errors
- Pulled all current files from main
- Fixed routes for current pages
- Setup basic backend server with sign in and sign up routes
- Setup sign up method in SignUp.js (called when SIGN UP button is pressed)
- Setup sign in method in SignIn.js (called when LOGIN button is pressed)
- Made updates to my CSS page
- Should be able to be safely merged now
- Updated signin and signup .js files names to PascalCase

Tasks titles:
- Setup signup route
- Setup login route
- Create Routes
- Setup backend server

youtube video link below:
https://youtu.be/oUHcSGoQLCo
